edx_rest_framework_extensions/settings.py

```diff
@@ -81,3 +84,10 @@ def get_jwt_issuers():
         return jwt_issuers
     # If we do not, return the deprecated configuration
     return _get_deprecated_jwt_issuers()
+
+
+def get_first_jwt_issuer():
+    """
+    Retrieves the first issuer in the JWT_ISSUERS list.
```

Comment:
Could you add a comment about why `first` is ok?  Do we need to say when this would go away and link to a ticket?